CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # This file is described here:  https://help.github.com/en/articles/about-code-owners
 
 # Global Owners: These are brigadecore org maintainers + specific outside collaborators
-* @brigadecore/maintainers @a-muk @shubham1172
\ No newline at end of file
+* @brigadecore/maintainers @a-muk @shubham1172
```

Comment:
fwiw, the "errors" here are my problem.

It's complaining that either that team does not exist or doesn't have write access to this repo and neither of those things are true. It may be a bug in GitHub itself and this same issue is affecting other repos as well.